### ISSUE TITLE
[Blood Hallway Door] Door Bugged if Adds Pulled Up Stairs Before  

### ISSUE THREAD

**Issue by Bartok14** [WoWManiaUK/Redemption#5927]
**Links:**

**What is Happening:**
In the Blood Hallway leading to BPC in ICC if you pull the adds up the stairs to the door before it opens it won't open and will stay closed. 

**What Should happen:**
Not that. I'm assuming the adds on the other side of the door might be bugged and agro and that's why the door gets bugged but I don't really know. You will have to test. Possible fix might be to have the adds unengageable until the door is opened.

**Comment by Rushor** [comment]
Can you make a quick video on how to reproduce or show the npcs that get pulled in order to bug it

**Comment by Bartok14** [comment]
I can't make a video unfortunately. But its the adds that as soon as you enter the Blood Hallway. I was tanking adds yesterday and the other tank pulled them off of me. Before they were dead I went up the stairs towards where the blood prince council is. This was before the adds in the previous room were dead. The door wouldn't open after that and we had to get GM assistance.

**Comment by svijetlo** [comment]
it is old bug resurfacing again if adds are killed "fast" and on stairs leading to BPQ room, doors it will not trigger  open

**Comment by GMZinius** [comment]
> it is old bug resurfacing again if adds are killed "fast" and on stairs leading to BPQ room, doors it will not trigger open

Thank you @svijetlo for info, is this the old report?
https://github.com/WoWManiaUK/Redemption/issues/4498

**Comment by svijetlo** [comment]
> > it is old bug resurfacing again if adds are killed "fast" and on stairs leading to BPQ room, doors it will not trigger open
> 
> Thank you @svijetlo for info, is this the old report? #4498

yes that is the one , also can not remember post but this issue was posted before if trash is pulled close to doors and killed of there doors will not open

**Comment by GMZinius** [comment]
Confirmed bug @Keader 

If you fight and kill Darkfallen adds and the Darkfallen Advisor https://www.wow-mania.com/armory/?npc=37571 on the stairs then the door bugs

If you fight the adds near or around the Empowering blood Orb its fine

**Comment by svijetlo** [comment]
> Confirmed bug @Keader
> 
> If you fight and kill Darkfallen adds and the Darkfallen Advisor https://www.wow-mania.com/armory/?npc=37571 on the stairs then the door bugs at the Crimson Hall
> 
> ![image](https://user-images.githubusercontent.com/108176634/194765519-e581bd9a-71d8-49fb-a7c6-5c46da83e379.png)
> 
> If you fight the adds near or around the Empowering blood Orb its fine
> 
> ![image](https://user-images.githubusercontent.com/108176634/194765465-97a55334-ec08-4240-b775-4624db940681.png)

yap, that is it trigger for doors is only around orb

**Comment by Bartok14** [comment]
> Confirmed bug @Keader
> 
> If you fight and kill Darkfallen adds and the Darkfallen Advisor https://www.wow-mania.com/armory/?npc=37571 on the stairs then the door bugs at the Crimson Hall
> 
> ![image](https://user-images.githubusercontent.com/108176634/194765519-e581bd9a-71d8-49fb-a7c6-5c46da83e379.png)
> 
> If you fight the adds near or around the Empowering blood Orb its fine
> 
> ![image](https://user-images.githubusercontent.com/108176634/194765465-97a55334-ec08-4240-b775-4624db940681.png)

Could you also check the frost wing as well to see if this is an issue? I have also experienced it where the door to Valithria refused to open after killing the adds. I'm wondering if this is the same issue happening over there with the adds from the hallway fight.

**Comment by GMZinius** [comment]
> Could you also check the frost wing as well to see if this is an issue? I have also experienced it where the door to Valithria refused to open after killing the adds. I'm wondering if this is the same issue happening over there with the adds from the hallway fight.

Need new report for that issue @Bartok14 please

**Comment by svijetlo** [comment]
> > Could you also check the frost wing as well to see if this is an issue? I have also experienced it where the door to Valithria refused to open after killing the adds. I'm wondering if this is the same issue happening over there with the adds from the hallway fight.
> 
> Need new report for that issue @Bartok14 please

reopen this post https://github.com/WoWManiaUK/Redemption/issues/5677   
this bugs is back on live i seen it happen 2 times in last week

**Comment by Keader** [comment]
try now pls

**Comment by GMZinius** [comment]
Thank you @Keader 
thats seems fine 👍 

![image](https://user-images.githubusercontent.com/108176634/200218999-ce7a5731-94f6-40a8-9d31-ba25fa449f66.png)
![image](https://user-images.githubusercontent.com/108176634/200219097-32359524-0efa-424e-a92d-0b520c0f219b.png)
